README.md

```diff
@@ -67,6 +67,10 @@ are two workarounds:
     Note however that the debugger must be running/listening as otherwise the JVM will emit a connection 
     refused error.
 
+## Projects verified with Gobra
```

Comment:
Just some more suggestions for alternative titles: 

- Ongoing and completed verification projects involving Gobra
- Verification projects involving Gobra